controllers/widget.js

```diff
@@ -285,13 +284,14 @@ function _onTouchend(e) {
         _applyOuterProperties(_properties.defaultStyle);
         _applyInnerProperties(_properties.defaultStyle);
     }
+}
 
-    if (e.type === 'touchend') {
-        $.trigger('click', {
-            type: "click",
-            source: $
-        });
-    }
+function _onClick(e) {
+	var event = {
+		type: 'click',
+		source: $
+	};
+	$.fireEvent('click', event);	
```

Comment:
fireEvent is a reference to trigger, so that no need to use that:

https://github.com/FokkeZB/nl.fokkezb.button/blob/master/controllers/widget.js#L345